lib/Dialect/FIRRTL/Transforms/GrandCentral.cpp

```diff
@@ -1850,6 +1865,8 @@ void GrandCentralPass::runOnOperation() {
           AnnotationSet::removeAnnotations(op, [&](Annotation annotation) {
             if (!annotation.getClass().startswith(viewAnnoClass))
               return false;
+            auto isNonlocal = annotation.getMember<FlatSymbolRefAttr>(
+                                  "circt.nonlocal") != nullptr;
```

Comment:
`hasMember<FlatSymbolRefAttr>("circt.nonlocal")` should work!

Reply:
I couldn't find the `Annotation::hasMember` , I recall something similar existing, was it removed ?